<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjoutInteractionWindow</class>
 <widget class="QWidget" name="AjoutInteractionWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter une interaction</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="1">
        <widget class="QTextEdit" name="newEventText"/>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="existingEventButton">
         <property name="text">
          <string>Ajouter une interaction existante</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="newEventButton">
         <property name="text">
          <string>Ajouter une nouvelle interaction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quitButton">
         <property name="text">
          <string>Quitter</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
